device-types/HPE/FlexNetwork-5520-24G-SFP-4SFPP-HI.yaml

```diff
@@ -0,0 +1,100 @@
+---
+manufacturer: HPE
+model: FlexNetwork 5520-24G-SFP-4SFPP HI
+slug: hpe-flexnetwork-5520-24g-sfp-4sfpp-hi
+part_number: R8M27A
+u_height: 1
+is_full_depth: false
+weight: 6.6
+weight_unit: kg
+comments: '[HPE FlexNetwork 5520 HI Switch Series](https://www.hpe.com/psnow/doc/a50004733enw?jumpId=in_psnow_23944961-f82d-48ce-9f08-ec7c755462e0_gaiw)'
+console-ports:
+  - name: Console
+    type: rj-45
+# dual personality console
+#   - name: Console
+#     type: usb-mini-b
+module-bays:
+  - name: port expansion module slot1
+    type: slot1
+  - name: PSU1
+    position: PSU1
```

Comment:
Does the PSU module have `PSU{module}`? If so, you probably want to adjust this to just be `1`